classes/LocalizationPack.php

```diff
@@ -352,7 +352,13 @@ protected function _installCurrencies($xml, $install_mode = false)
                         return false;
                     }
                     Cache::clear();
+                    // set default data from CLDR
                     $this->setCurrencyCldrData($currency, $defaultLang);
+
+                    // Following is required when installing new currency on existing languages:
+                    // we want to properly update the symbol in each language
+                    $localeRepoCLDR = $this->getCldrLocaleRepository();
+                    $currency->refreshLocalizedCurrencyData(Language::getLanguages(), $localeRepoCLDR);
```

Comment:
It's sad that we add this public method for this.. Even though it's a good thing to perform this modif in a single place, though you need to inject the repo in it Maybe it should be a service which makes this update rather than the `Currency` itself

Reply:
I think it could be done in next step for CLDR (currency/language cutomization)
this method is a best effort to properly update currency data from CLDR.

Reply:
more precisely : in this version the method override currency_lang data
in next version with customization it will not, we will have to find a better way to update data

Reply:
well in the next version it will still update the data if they are not present right?
but ok for an improvement in next version